crawler/config/centres.py

```diff
@@ -239,3 +263,42 @@
         CENTRE_KEY_SKIP_UNCONSOLIDATED_SURVEILLANCE_FILES: False,
     },
 ]
+
+
+def get_centres_config(config: Config, data_source: str = "") -> List[CentreConf]:
+    """Get the centres config from MongoDB. If MongoDB does not contain any centres config, it will become populated
+    with the values in the app config for centres.
+
+    Arguments:
+        config {Config}: The configuration object for the whole application.
+        data_source {str}: The data source filter to apply to centre configs, or None to apply no filter.
```

Comment:
if data_source only admits a limited range of values (rabbitmq and sftp), it would be better typed if defined as enum

Reply:
I did do that, but it doesn't serialise to MongoDB cleanly when I use that.